faker/providers/date_time/__init__.py

```diff
@@ -1172,13 +1172,15 @@ class Provider(BaseProvider):
 
     regex = re.compile(timedelta_pattern)
 
-    def unix_time(self, end_datetime=None):
+    def unix_time(self, start_datetime=None, end_datetime=None):
```

Comment:
I'm a little worried about backward compatibility, in case people have started to use `unix_time` with positional argument.

I would change the signature to be `unix_time(self, end_datetime=None, start_datetime=None)`.